metrics/director.go

```diff
@@ -82,10 +82,10 @@ var (
 		Help: "The total stat queries the director issues. The status can be Succeeded, Cancelled, Timeout, Forbidden, or UnknownErr",
 	}, []string{"server_name", "server_url", "server_type", "result", "cached_result"}) // result: see enums for DirectorStatResult
 
-	PelicanDirectorServerCount = promauto.NewCounterVec(prometheus.CounterOpts{
+	PelicanDirectorServerCount = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "pelican_director_server_count",
-		Help: "Total number of servers, delineated by pelican/non-pelican and origin/cache",
-	}, []string{"server_name", "server_type", "server_url", "server_web_url", "server_lat", "server_long", "from_topology"})
+		Help: "The number of servers, delineated by pelican/non-pelican and origin/cache",
```

Comment:
Let's be even more explicit and say "The number of servers currently recognized by the Director, delineated by blah blah"